torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -953,26 +951,39 @@ void LLVMCodeGen::visit(const Intrinsics* v) {
         return;
       } break;
 
-#define UNARY_MATH_CASE(enum, name, type)                             \
-  case enum: {                                                        \
-    auto callee = module_->getOrInsertFunction(                       \
-        name, llvm::FunctionType::get(type, {type}, false), {});      \
-    call_ty = callee.getFunctionType();                               \
-    call_fn = callee.getCallee();                                     \
-    applyMathFunctionAttributes(llvm::cast<llvm::Function>(call_fn)); \
+#define SIMD_UNARY_MATH_CASE(enum, name, type)                               \
+  case enum: {                                                               \
+    llvm::FunctionCallee callee;                                             \
+    std::string fname;                                                       \
+    if (v->dtype().lanes() == 4) {                                           \
+      if (v->op_type() == kErfc) {                                           \
```

Comment:
Having this check is a little gross, but let's see how it scales as we add more Sleef calls.  We might end up wanting to refactor this another way, maybe passing the Sleef name into the macro.